sources/lib/api/gcp/stackdriver/monitoring/_timeseries.py

```diff
@@ -14,49 +14,54 @@
 
 """Provides access to metric data as pandas dataframes."""
 
+import gcloud.monitoring
 import pandas
 
 import gcp
-from . import _impl
 from . import _utils
 
 
-class Query(_impl.Query):
+class Query(gcloud.monitoring.Query):
```

Comment:
Retaining the fork of the base library in `impl_` was the only viable option earlier, but `gcloud.monitoring` was finally published in PyPI on Friday.